### DISCUSSION TITLE
Missing Cover Art

### DISCUSSION THREAD

**Discussion by zipzagster** [music-assistant/support#2931] [Q&A]
I have a wall tablet running Fully that needs to pull Album Art from the MA server, as it doesn't have internet access. This worked fine until a MA upgrade a few months ago. Now all the album art is all blank.  I can no longer find a setup option where I tell the frontend to only get album art from the server. Help?

**Comment by OzGav** [comment]
That is the standard now that artwork is routed through the server. What happens if you try and manually download the images?

**Comment by zipzagster** [comment]
The album art is being pulled from tidal, not the server. The tablet doesn't have internet access.

https://resources.tidal.com/images/febc7760/486a/4634/87ac/a0a379725847/750x750.jpg
